test/neura/interpreter/phiOp_interpret.mlir

```diff
@@ -0,0 +1,20 @@
+// RUN: neura-interpreter %s | FileCheck %s
+
```

Comment:
Can we run mlir-neura-opt with `--assign-accelerator --leverage-predicated-value` so the IR would contain `PredicatedValue` as https://github.com/coredac/dataflow/blob/64e7b3537cc5de007ec81b706c67cb20f0ce2860/test/neura/ctrl/branch_no_arg.mlir#L62
and then run `neura-interpreter` again to double check?